packages/files/README.md

```diff
@@ -0,0 +1,18 @@
+# `@shopify/files`
+
+[![Build Status](https://travis-ci.org/Shopify/quilt.svg?branch=master)](https://travis-ci.org/Shopify/quilt)
+[![License: MIT](https://img.shields.io/badge/License-MIT-green.svg)](LICENSE.md) [![npm version](https://badge.fury.io/js/%40shopify%2Ffiles.svg)](https://badge.fury.io/js/%40shopify%2Ffiles.svg) [![npm bundle size (minified + gzip)](https://img.shields.io/bundlephobia/minzip/@shopify/files.svg)](https://img.shields.io/bundlephobia/minzip/@shopify/files.svg)
+
+## Installation
+
+```bash
+$ yarn add @shopify/files
+```
+
+## Usage
+
+This package exposes utilities and enums related to handling files.
```

Comment:
handling files in what way? Reading of the file? Writing of the file? What type of files? Is this a file a a consumer are using in the application, a file a end user uploaded?

Reply:
It would be to get MIME types from file names, or get file extensions from MIME types - those are the utilities exposed.

Currently it's mostly supporting media files, but https://github.com/Shopify/quilt/pull/1372 would ideally follow for other more common file types - this would fix https://github.com/Shopify/quilt/issues/411

I've updated the `README.md`, let me know if that's a bit clearer for consumers